PublicMapping/districtbuilder
# Add evaluate mode view for competitiveness

## Overview

- Adds a newly computed metric for Competitiveness, computed as the average PVI for all districts that have population
- Adds a new page for Competitiveness metric including a map and sidebar
- Add a dropdown to switch between 2016 / 2020 / combined PVI

### Checklist


### Demo

![image](https://user-images.githubusercontent.com/66973361/122587151-7df03f00-d02b-11eb-8bde-f1eda51cda77.png)


### Notes

Per my Slack conversation earlier with @maurizi , there is still some ambiguity about whether we want to allow a switch for PVI / election margin to display both metrics here. These will likely be very similar, 

## Testing Instructions

- Create a map in a region that has voting data
- Create districts
- Go to evaluate mode - Competitiveness and review. Expect the PVI computations to match those included in the Project Sidebar

Closes #810
